curriculum/challenges/english/blocks/lecture-working-with-higher-order-functions-and-callbacks/67329ffd75010f5ddeb4ea13.md

```diff
@@ -7,7 +7,7 @@ dashedName: what-is-a-callback-function-and-how-does-it-work-with-the-foreach-me
 
 # --interactive--
 
-In JavaScript, a callback function is a function that is passed as an argument to another function, so that the outer function can invoke it at a specific point.
+Imagine you have a function that performs a task, and you want to make its behavior more flexible. Instead of writing everything inside one function, you can pass another function (a callback) as an argument. The function that receives the callback decides when to run it — this might happen immediately, later, or even asynchronously. This makes your code more modular and easier to customize.
```

Comment:
this paragraph was properly updated in the previous PR, the paragraph to update is lower, "Imagine you have a function that performs a task, and you want to do something after that task is complete."